src/App.tsx

```diff
@@ -628,48 +643,48 @@ function App() {
             <div className="app">
                 <SimulariumContext.Provider
                     value={{
-                        trajectoryName,
-                        productName,
                         adjustableAgentName,
                         currentProductionConcentration:
                             liveConcentration[productName] || 0,
                         fixedAgentStartingConcentration:
                             inputConcentration[AgentName.A] || 0,
+                        getAgentColor: simulationData.getAgentColor,
```

Comment:
i added a few things to the context but I also alphabetized them, which is why there are so many changes